src/main/java/uk/gov/hmcts/darts/event/service/EventProcessor.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.darts.event.service;
+
+import java.util.List;
+
+public interface EventProcessor {
+    List<Integer> processCurrentEvent();
```

Comment:
shall we use more specific name? In the future we might introduce other types of events processors. 
We could call this for example `CleanupCurrentFlagEventProcessor` or sim